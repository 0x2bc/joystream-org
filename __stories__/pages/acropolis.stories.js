import React from 'react';
import { storiesOf } from '@storybook/react';

import AcropolisPage from '../../src/pages/acropolis';

storiesOf('Pages|Acropolis', module).add('default', () => <AcropolisPage />);
