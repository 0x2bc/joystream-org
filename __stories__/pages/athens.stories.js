import React from 'react';
import { storiesOf } from '@storybook/react';

import AthensPage from '../../src/pages/athens';

storiesOf('Pages|Athens', module).add('default', () => <AthensPage />);
