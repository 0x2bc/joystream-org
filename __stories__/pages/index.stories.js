import React from 'react';
import { storiesOf } from '@storybook/react';

import IndexPage from '../../src/pages/index';

storiesOf('Pages|Home', module).add('default', () => <IndexPage />);
