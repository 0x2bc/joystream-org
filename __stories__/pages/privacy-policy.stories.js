import React from 'react';
import { storiesOf } from '@storybook/react';

import PrivacyPolicyPage from '../../src/pages/privacy-policy';

storiesOf('Pages|Privacy Policy', module).add('default', () => <PrivacyPolicyPage />);
