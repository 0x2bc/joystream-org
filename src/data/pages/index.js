import { ReactComponent as ValidatorsImage } from '../../assets/svg/active-validators.svg';
import { ReactComponent as StorageImage } from '../../assets/svg/storage.svg';
import { ReactComponent as MemberImage } from '../../assets/svg/council-member.svg';
import { ReactComponent as ScreenerImage } from '../../assets/svg/screener.svg';
import { ReactComponent as MembershipCuratorImage } from '../../assets/svg/membership-curator.svg';
import { ReactComponent as ContentCuratorImage } from '../../assets/svg/content-curator.svg';
import { ReactComponent as ContentCreatorImage } from '../../assets/svg/content-creator.svg';
import { ReactComponent as BandwidthProviderImage } from '../../assets/svg/bandwidth-provider.svg';
import { ReactComponent as DiscoveryProviderImage } from '../../assets/svg/discovery-provider.svg';
import { ReactComponent as LiveStreamingProviderImage } from '../../assets/svg/live-streaming-provider.svg';
import { ReactComponent as BuilderImage } from '../../assets/svg/builder.svg';
import { ReactComponent as CommunicationModeratorImage } from '../../assets/svg/communication-moderator.svg';

const roles = {
  active: [
    {
      image: ValidatorsImage,
      title: 'Validator',
      to: '/roles#Validator',
      key: 'validatorsCount',
      hasLabel: false,
    },
    {
      image: MemberImage,
      title: 'Council Member',
      to: '/roles#Council-Member',
      key: 'councilMembersCount',
      hasLabel: false,
    },
    {
      image: StorageImage,
      title: 'Storage Provider',
      to: '/roles#Storage-Provider',
      key: 'storageProviders',
      hasLabel: true,
    },
  ],
  future: [
    { image: ScreenerImage, title: 'Membership Screener', to: '/roles' },
    {
      image: MembershipCuratorImage,
      title: 'Membership Curator',
      to: '/roles',
    },
    { image: ContentCuratorImage, title: 'Content Curator', to: '/roles' },
    { image: ContentCreatorImage, title: 'Content Creator', to: '/roles' },
    {
      image: BandwidthProviderImage,
      title: 'Bandwidth Provider',
      to: '/roles',
    },
    {
      image: DiscoveryProviderImage,
      title: 'Discovery Provider',
      to: '/roles',
    },
    {
      image: LiveStreamingProviderImage,
      title: 'Live Streaming Provider',
      to: '/roles',
    },
    { image: BuilderImage, title: 'Builder', to: '/roles' },
    {
      image: CommunicationModeratorImage,
      title: 'Communication Moderator',
      to: '/roles',
    },
  ],
};

export { roles };
