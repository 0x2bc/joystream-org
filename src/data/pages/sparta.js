import React from 'react';
import Link from '../../components/Link';
import { ReactComponent as ValidatorsImage } from '../../assets/svg/active-validators.svg';
import { ReactComponent as MemberImage } from '../../assets/svg/council-member.svg';
import { ReactComponent as payoutImage } from '../../assets/svg/participation-payout.svg';
import { ReactComponent as blockImage } from '../../assets/svg/block-platform-content-files.svg';
import { ReactComponent as emblemImage } from '../../assets/svg/memberships.svg';

const roles = {
  active: [
    {
      image: ValidatorsImage,
      title: 'Validator',
      to: '/roles#Validator',
      key: 'validatorsCount',
      hasLabel: true,
      type: 'most',
    },
    {
      image: MemberImage,
      title: 'Council Member',
      to: '/roles#Council-Member',
      key: 'councilMembersCount',
      hasLabel: true,
      type: 'most',
    },
  ],
};

const analytics = [
  { title: 'Participation Payout', image: payoutImage, value: '$1576' },
  {
    title: 'Active Validators',
    image: ValidatorsImage,
    key: 'validatorsCount',
  },
  { title: 'Memberships', image: emblemImage, key: 'membershipsMembers' },
  { title: 'Block Height', image: blockImage, key: 'blockHeight' },
];

const goals = [
  {
    title: 'Build and release a blockchain with a working Council',
    text: (
      <>
        As a "user governed media platform", allowing users to elect{' '}
        <Link to="/roles#Council-Member">Council Members</Link> to represent
        their interest in day to day operations was a major goal for us.
      </>
    ),
  },
  {
    title:
      'Keep the quantity of more or less critical bugs at an acceptable level',
    text:
      'As demonstrated by the fact that the network crashed, we did not reach our goals in this case.',
    state: 'postponed',
  },
];

export { roles, analytics, goals };
