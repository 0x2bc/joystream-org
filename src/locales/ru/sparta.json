{
  "sparta": {
    "siteMetadata": {
      "description": "Изучаем тестнет Sparta"
    },
    "hero": {
      "title": "Сеть Sparta",
      "text": "Изучайте доступные роли и выбирайте наиболее подходящие вам. Влияйте на развитие платформы, зарабатывая при этом реальные деньги.",
      "chipText": "Что это?"
    },
    "heroCard": {
      "title": "ОТКАЗ СЕТИ",
      "markdown": "\n ### ОБНАРУЖЕНА НЕИСПРАВНОСТЬ\n\n  В пятницу, 29 марта, сеть Sparta вышла из строя из-за [известного бага в Substrate](https://github.com/paritytech/substrate/pull/2130) который не был устранен перед выпуском.\n\n  Подробная информация [в блоге](https://blog.joystream.org/sparta-sacked/)."
    },
    "modal": {
      "title": "Шлем Спарты",
      "text": "<0>Спартанский армейский шлем</0> олицетворяет то, чем сейчас наиболее известна Спарта, ее военную мощь. Для своего времени она также была известна своей выдающейся системой управления."
    },
    "testnetGoals": {
      "title": "Цели тестнета",
      "subtitle": "Цели, приведенные ниже, являются упрощенным представлением ключевых результатов, перечисленных в нашем релизе OKR.",
      "goals": {
        "blockchain": {
          "title": "Построение и релиз блокчейна с работающим Советом",
          "text": "В качестве \"медиа-платформы, управляемой пользователями\", наша главная цель - предоставить пользователям возможность избирать <0>членов Совета</0>, которые будут представлять интересы пользователей в повседневных действиях."
        },
        "bugs": {
          "title": "Поддержка критических ошибок на приемлемом уровне",
          "text": "Как показывает факт выхода нашей сети из строя, мы не достигли этой цели."
        }
      }
    },
    "roles": {
      "title": "Поощряемые роли в сети Sparta"
    },
    "map": {
      "title": "Город Sparta",
      "text": "<0>Спарта была выдающимся городом-государством в Древней Греции.</0> Помимо своей военной мощи, она была уникальной среди своих греческих соперников благодаря  четко определенной социальной системе и конституции. <1/><1/> Мы выбрали название \"Спарта\" из-за ее исторического значения на пути к демократии и верховенству закона, каким бы суровым по современным меркам он ни был."
    }
  }
}
