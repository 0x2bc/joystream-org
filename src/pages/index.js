import React from 'react';
import { pagePropTypes } from '../propTypes';

import getApiPath from '../utils/getApiPath';
import mapStatusDataToAnalytics from '../utils/mapStatusDataToAnalytics';
import mapStatusDataToRoles from '../utils/mapStatusDataToRoles';

import withApi from '../components/_enhancers/withApi';

import BaseLayout from '../components/_layouts/Base';
import Link from '../components/Link';
import Subheader from '../components/Subheader';
import TestnetItem from '../components/TestnetItem';
import Analytics from '../components/Analytics';
import Button from '../components/Button';
import TitleWrapper from '../components/TitleWrapper';
import RoleList from '../components/RoleList';
import ColumnsLayout from '../components/ColumnsLayout';
import LayoutWrapper from '../components/LayoutWrapper';
import Hero from '../components/Hero';
import SiteMetadata from '../components/SiteMetadata';

import AcropolisImage from '../assets/svg/acropolis-main.svg';
import heroImage from '../assets/svg/hero-builder.svg';
import RomeImage from '../assets/svg/rome-main.svg';
import constantinopleImage from '../assets/svg/constantinople-main.svg';
import alexandriaImage from '../assets/svg/alexandria-main.svg';
import { ReactComponent as TickImage } from '../assets/svg/tick.svg';
import { ReactComponent as ClockImage } from '../assets/svg/clock.svg';

import { roles } from '../data/pages';
import { launchDate as alexandriaNetworkLaunchDate } from '../data/pages/alexandria';

import './style.scss';

const activeTestnet = {
  name: 'Alexandria',
  incentivesLink: 'https://www.joystream.org/testnet',
};

const IndexPage = ({ content }) => (
  <BaseLayout>
    <SiteMetadata
      title="Joystream: The video platform DAO"
      description="Joystream is a video platform controlled, owned, and operated by its users."
    />
    <Hero
      image={heroImage}
      title={
        <>
          The video <br /> platform DAO
        </>
      }
      animationStartValue={0}
    >
      <p className="IndexPage__hero-paragraph">
        Joystream is a video platform controlled, owned, and operated by its users.
      </p>
      <div className="IndexPage__hero-group">
        <Button noWrap large secondary className="IndexPage__hero-button" href="/testnet/">
          Earn Tokens
        </Button>
        <Button
          noWrap
          large
          secondary
          reversed
          className="IndexPage__hero-button"
          href="https://testnet.joystream.org/"
        >
          Launch UI
        </Button>
      </div>
    </Hero>

    <LayoutWrapper>
      <TestnetItem
        title="Alexandria Testnet"
        image={alexandriaImage}
        children={
          <>Alexandria is our seventh testnet, introducing a variety of important technical improvements.</>
        }
        button={{
          label: 'Explore Alexandria',
          to: '/alexandria',
        }}
      />
    </LayoutWrapper>

    <Analytics title="Testnet Metrics" large content={mapStatusDataToAnalytics(content)}>
      <Button secondary href={activeTestnet.incentivesLink}>
        Participate and Earn Monero
      </Button>
    </Analytics>

    {/*

    <LayoutWrapper>
      <TestnetItem
        title="Alexandria Testnet"
        image={alexandriaImage}
        children="Alexandria is our seventh testnet, focused on a variety of important technical improvements."
        date={alexandriaNetworkLaunchDate}
        button={{
          label: 'Explore Alexandria',
          to: '/alexandria',
        }}
      />
    </LayoutWrapper>

    */}

    <LayoutWrapper dark>
      <TitleWrapper title="Become a user, run the platform">
        <Subheader
          title="Active roles on the current testnet"
          content="learn more, join a role and subscribe for more"
          icon={TickImage}
        />
        <ColumnsLayout>
          <RoleList roles={roles.active} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>
        <Subheader
          title="Roles coming in future testnets"
          content="choose a role, learn more and subscribe to get updated"
          icon={ClockImage}
        />
        <ColumnsLayout columnsCount={3}>
          <RoleList roles={roles.future} content={mapStatusDataToRoles(content)} />
        </ColumnsLayout>

        <Button secondary to="/roles" className="IndexPage__roles-button">
          Learn more about the roles
        </Button>
      </TitleWrapper>
    </LayoutWrapper>
  </BaseLayout>
);

IndexPage.propTypes = pagePropTypes;

export { IndexPage };
export default withApi(IndexPage, getApiPath('/'));
