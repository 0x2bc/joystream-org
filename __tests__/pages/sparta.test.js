import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import { SpartaPage } from '../../src/pages/sparta';

import testContent from '../../__mocks__/data/testContent';

configure({ adapter: new Adapter() });

describe('SpartaPage page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SpartaPage content={testContent} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
