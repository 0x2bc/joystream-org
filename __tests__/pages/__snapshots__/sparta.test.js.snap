// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpartaPage page renders correctly 1`] = `
<div>
  <nav
    className="Navbar"
  >
    <div
      className="Navbar__container"
    >
      <a
        className="Link"
        href="/"
      >
        <icon-mock
          className="Navbar__logo"
        />
      </a>
      <div
        className="Navbar__links"
      >
        <a
          className="Navbar__link Navbar__link--highlighted Link Link--highlighted"
          href="https://www.jsgenesis.com/#openings"
          label="We are hiring!"
        >
          We are hiring!
        </a>
        <a
          className="Navbar__link Link"
          href="https://blog.joystream.org/"
          label="Blog"
        >
          Blog
        </a>
        <a
          className="Navbar__link Link"
          href="https://blog.joystream.org/manifesto/"
          label="Manifesto"
        >
          Manifesto
        </a>
        <a
          className="Navbar__link Link"
          href="https://github.com/Joystream/whitepaper/blob/master/paper.pdf"
          label="Whitepaper"
        >
          Whitepaper
        </a>
        <a
          className="Navbar__link Link"
          href="/roles"
          label="Roles"
        >
          Roles
        </a>
        <a
          className="Navbar__link Link"
          href="https://github.com/Joystream/joystream"
          label="Repository"
        >
          Repository
        </a>
      </div>
      <div
        className="Navbar__trigger"
        onClick={[Function]}
        role="presentation"
      />
    </div>
  </nav>
  <section
    className="Hero Hero--indented"
  >
    <div
      className="Hero__container"
    >
      <div
        className="Hero__content"
      >
        <h1
          className="Hero__title"
        >
          Sparta Network
        </h1>
        <div
          className="Hero__description"
        >
          <p
            className="SpartaPage__hero-paragraph"
          >
            Explore available roles and pick the one that suits you the most. Influence platforms development earning Monero in the process.
          </p>
          <div
            className="Card Card--error"
          >
            <p
              className="Card__header"
            >
              <icon-mock
                className="Card__icon"
              />
              Network down
            </p>
            <div
              className="Card__content"
            >
              <div>
                <h3>
                  FAILURE IDENTIFIED
                </h3>
                <p>
                    On Friday the 29th of March, the Sparta network went down due to a 
                  <a
                    href="https://github.com/paritytech/substrate/pull/2130"
                  >
                    known bug in substrate
                  </a>
                   that we had not pulled down before release.
                </p>
                <p>
                    More details can be found in this 
                  <a
                    href="https://blog.joystream.org/sparta-sacked/"
                  >
                    blog post
                  </a>
                  .
  
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="Hero__image-wrapper"
      >
        <button
          className="Chip"
          onClick={[Function]}
        >
          <icon-mock
            className="Chip__icon"
          />
          <span
            className="Chip__text"
          >
            What is this?
          </span>
        </button>
        <div
          className="Plx  "
          disabled={false}
          style={
            Object {
              "visibility": "hidden",
            }
          }
        >
          <icon-mock
            className="Hero__image"
          />
        </div>
      </div>
    </div>
  </section>
  <div
    className="LayoutWrapper"
  >
    <div
      className="LayoutWrapper__container"
    >
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Network Statistics
        </h2>
        <div
          className="TitleWrapper__subtitle"
        >
          As they were at the end of the network.
        </div>
        <section
          className="Analytics"
        >
          <div
            className="Analytics__container"
          >
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Participation Payout
              </p>
              <p
                className="AnalyticsItem__value"
              >
                $1576
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Active Validators
              </p>
              <p
                className="AnalyticsItem__value"
              >
                20
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Memberships
              </p>
              <p
                className="AnalyticsItem__value"
              >
                164
              </p>
            </div>
            <div
              className="AnalyticsItem"
            >
              <icon-mock
                className="AnalyticsItem__image"
              />
              <p
                className="AnalyticsItem__title"
              >
                Block Height
              </p>
              <p
                className="AnalyticsItem__value"
              >
                1110117
              </p>
            </div>
          </div>
        </section>
      </section>
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Testnet Goals
        </h2>
        <div
          className="TitleWrapper__subtitle"
        >
          The goals below is a simplified and qualitative representation of the Key Results listed in our release OKR.
        </div>
        <div
          className="Animated"
        >
          <div
            className="GoalItem"
          >
            <div
              className="GoalItem__icon-container"
            >
              <icon-mock
                className="GoalItem__icon"
              />
              <p
                className="GoalItem__status GoalItem__status--achieved"
              >
                Achieved
              </p>
            </div>
            <div
              className="GoalItem__content"
            >
              <p
                className="GoalItem__title"
              >
                Build and release a blockchain with a working Council
              </p>
              As a "user governed media platform", allowing users to elect
               
              <a
                className="Link"
                href="/roles#Council-Member"
              >
                Council Members
              </a>
               to represent their interest in day to day operations was a major goal for us.
            </div>
          </div>
        </div>
        <div
          className="Animated"
        >
          <div
            className="GoalItem"
          >
            <div
              className="GoalItem__icon-container"
            >
              <icon-mock
                className="GoalItem__icon"
              />
              <p
                className="GoalItem__status GoalItem__status--postponed"
              >
                Postponed
              </p>
            </div>
            <div
              className="GoalItem__content"
            >
              <p
                className="GoalItem__title"
              >
                Keep the quantity of more or less critical bugs at an acceptable level
              </p>
              As demonstrated by the fact that the network crashed, we did not reach our goals in this case.
            </div>
          </div>
        </div>
      </section>
    </div>
  </div>
  <div
    className="LayoutWrapper LayoutWrapper--dark"
  >
    <div
      className="LayoutWrapper__container"
    >
      <section
        className="TitleWrapper"
      >
        <h2
          className="TitleWrapper__title"
        >
          Roles available on the Sparta testnet
        </h2>
        
        <div
          className="ColumnsLayout"
          data-columns={2}
        >
          <a
            className="RoleCard RoleCard--labeled Link"
            href="/roles#Validator"
          >
            <icon-mock
              className="RoleCard__image"
            />
            <div
              className="RoleCard__content"
            >
              <p
                className="RoleCard__title"
              >
                Validator
              </p>
              <p
                className="RoleCard__info"
              >
                <icon-mock
                  className="RoleCard__icon"
                />
                At most, 
                <span
                  className="RoleCard__counter"
                >
                  20
                </span>
                 users occupied this role
              </p>
            </div>
          </a>
          <a
            className="RoleCard RoleCard--labeled Link"
            href="/roles#Council-Member"
          >
            <icon-mock
              className="RoleCard__image"
            />
            <div
              className="RoleCard__content"
            >
              <p
                className="RoleCard__title"
              >
                Council Member
              </p>
              <p
                className="RoleCard__info"
              >
                <icon-mock
                  className="RoleCard__icon"
                />
                At most, 
                <span
                  className="RoleCard__counter"
                >
                  12
                </span>
                 users occupied this role
              </p>
            </div>
          </a>
        </div>
      </section>
    </div>
  </div>
  <section
    className="MapInfo"
  >
    <div
      className="Map MapInfo__map"
      style={
        Object {
          "backgroundImage": "url(test-file-stub)",
        }
      }
    >
      <div
        className="Animated"
      >
        <icon-mock />
      </div>
      <p
        className="Map__info"
      >
        Map data ©
        2019
         Google
      </p>
    </div>
    <div
      className="MapInfo__content"
    >
      <div
        className="Animated"
      >
        <h1
          className="MapInfo__title"
        >
          The city of Sparta
        </h1>
        <p>
          <strong>
            Sparta was a prominent city-state in Ancient Greece.
          </strong>
           
          In addition to its military might, it was also unique both for its time and compared to its greek rivals due its defined social system and constitution.
          <br />
          <br />
          We chose the name Sparta due to it's historical significance in the path towards democracy and rule of law, however draconian by modern standard.
          <br />
          <br />
        </p>
      </div>
    </div>
  </section>
  <div
    className="CookiesNotice"
  >
    <div
      className="CookiesNotice__container"
    >
      <div
        className="CookiesNotice__content"
      >
        <p>
          This site uses cookies. By continuing to browse the site, you are agreeing to our use of cookies.
        </p>
        <p>
          <a
            className="CookiesNotice__link Link"
            href="/privacy-police"
          >
            Find out more ›
          </a>
        </p>
      </div>
      <button
        className="CookiesNotice__accept Button Button--secondary"
        onClick={[Function]}
        type="button"
      >
        Accept
      </button>
    </div>
  </div>
  <footer
    className="Footer"
  >
    <p
      className="Footer__header"
    >
      Get notified
    </p>
    <a
      className="Footer__button Button Button--small Button--light"
      href="http://joystream.us11.list-manage.com/subscribe/post?u=932de577aec9616d4516b4e0f&id=459ba8d1da"
      onClick={null}
    >
      Join the newsletter
    </a>
    <div
      className="Footer__socials"
    >
      <a
        className="Footer__social-wrapper"
        href="https://twitter.com/JoyStreamApp"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://github.com/Joystream"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://chat.joystream.org/home"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
      <a
        className="Footer__social-wrapper"
        href="https://t.me/joinchat/CNyeUxHD9H56m3e_44hXIA"
      >
        <icon-mock
          className="Footer__social-icon"
        />
      </a>
    </div>
    <div
      className="Footer__links"
    >
      <a
        className="Footer__link Footer__link--highlighted Link Link--highlighted"
        href="https://www.jsgenesis.com/#openings"
        label="We are hiring!"
      >
        We are hiring!
      </a>
      <a
        className="Footer__link Link"
        href="https://blog.joystream.org/"
        label="Blog"
      >
        Blog
      </a>
      <a
        className="Footer__link Link"
        href="https://blog.joystream.org/manifesto/"
        label="Manifesto"
      >
        Manifesto
      </a>
      <a
        className="Footer__link Link"
        href="https://github.com/Joystream/joystream"
        label="Repository"
      >
        Repository
      </a>
    </div>
  </footer>
</div>
`;
